### DISCUSSION TITLE
Should we check in `.astro-i18n/generated.d.ts`?

### DISCUSSION THREAD

**Discussion by reefdog** [Alexandre-Fernandez/astro-i18n#21] [Q&A]
Apologies, this reflects my ignorance about what purpose `.astro-i18n/generated.d.ts` serves, but: should I check this file into my repo, or add it to `.gitignore`?

**Comment by Alexandre-Fernandez** [comment]
> Apologies, this reflects my ignorance about what purpose `.astro-i18n/generated.d.ts` serves, but: should I check this file into my repo, or add it to `.gitignore`?

Sorry for the late reply, yes you can ignore everything inside `.astro-18n`. The folder contains generated content.
`generated.d.ts` is what provides the types/auto-completion for your translations, it's imported in `src/env.d.ts` and generated by the `sync` command.

**Reply by reefdog** [reply]
No worries at all on the reply timing. Thanks for responding and that's good to know, thanks!
